\documentclass{beamer}
\usetheme{Boadilla}
\setbeamertemplate{navigation symbols}{}
\setbeamertemplate{footline}{}

\title{Microarray Analysis}
\author{}
\date{}

\begin{document}
<<set-options, echo=FALSE, cache=FALSE>>=
opts_chunk$set(echo = FALSE, fig.path = 'figure/', dev = 'pdf', fig.align='center', fig.show='hold')
read_chunk('1_load.R')
@

<<load>>=
@

<<ext_code1>>=
read_chunk('1.5_functions.R')
@

<<func1>>=
@

<<ext_code2>>=
read_chunk('2_clean.R')
@

<<clean>>=
@

<<ext_code3>>=
read_chunk('3_functions.R')
@

<<func2>>=
@

<<ext_code4>>=
read_chunk('4_do.R')
@

\begin{frame}
    \titlepage
\end{frame}

\section{Microarray Workflow}
    \subsection{Samples}
        \begin{frame}
            % Experiment set up: 4 samples, a diagram of the thing in Nick's Lab book
        \end{frame}

% maybe change this bit into a single slide with a diagram?

    \subsection{Microarrays}
        \begin{frame}
            % Single channel Agilent microarray (or at least the data was collected using an agilent thing anyway)
            % to compare between arrays in the experiment normalised to 75th percentile and RMA
        \end{frame}
    
    \subsection{Data}
        \begin{frame}
            % I was given data based on the RNAseq experiments by Nick and Brian Lam
            % this was being used to look at expression differences between contigs that Brian made 
        \end{frame}
        
    \subsection{What I did}
        \begin{frame}
            % I also had the sequences of the contigs so I blasted the contigs against the mito
            % genome and used the contigs that had a perfect match to Eurydice genome as markers
            % of mitochondrial gene expression
        \end{frame}
        
\section{Graphs}
    \begin{frame}
        % whole timcourse plot
    \end{frame}
    
    \subsection{Circadian}
        \begin{frame}
            % whole circadian
        \end{frame}
        \begin{frame}
            % significant circadian
        \end{frame}    
    
    \subsection{tidal}
        \begin{frame}
            % whole tidal
        \end{frame}
        \begin{frame}
            % significant tidal
        \end{frame}        

\section{what I intend to do}
    \begin{frame}
        % now I have the probe sequences and raw data so I will do shit with them
    \end{frame}
    
\section{Immuno}
    \begin{frame}
        % pretty picture time
    \end{frame}


<<filter>>=
@

In perfect match group \Sexpr{rrnl_num} of the contigs match rrnL with an e value
of 0. The rest of the matches are shown below.

<<contig_table, results='asis'>>=
@

\clearpage
\section{Whole Timecourse}
\begin{figure}[h!]
    \caption{Mitochondrial gene expression: all time points}
<<whole_time_course>>=
@
\end{figure}
Can't actually show tidal time and circadian time on the same graph with these data

\clearpage
\section{Circadian}
\begin{figure}[h!]
    \caption{Mitochondrial gene expression: circadian time}
<<circadian_contig_graph>>=
@
\end{figure}
<<circa_t_test, results='asis'>>=
@
\begin{figure}[h!]
    \caption{Mitochondrial gene expression: circadian, significant change (p \textless 0.05)}
<<circa_filter_graph, fig.height=6>>=
@
\end{figure}

\clearpage
\section{Tidal}
\begin{figure}[h!]
    \caption{Mitochondrial gene expression: tidal time}
<<tidal_contig_graph>>=
@
\end{figure}
<<tidal_t_test,results='asis'>>=
@
\begin{figure}[h!]
    \caption{Mitochondrial gene expression: tidal, significant change (p \textless 0.05)}
<<tidal_filter_graph, fig.height=6>>=
@
\end{figure}
None of the changes in the perfect hits group are significant according to a Student t-test.
\clearpage
\section{Session information} 
<<session_info>>=
print(sessionInfo(), locale = FALSE)
@

\end{document}